api/v1beta1/cryostat_types.go

```diff
@@ -87,6 +87,25 @@ type CryostatSpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="JMX Connections Cache Options"
 	JmxCacheOptions *JmxCacheOptions `json:"jmxCacheOptions,omitempty"`
+	// Resource requirements for the Cryostat deployment
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:resourceRequirements"}
```

Comment:
This should be:
```suggestion
	// +operator-sdk:csv:customresourcedefinitions:type=spec
```
Having the descriptor here prevents the Core/DataSource/Grafana resources from showing in the UI.

Reply:
This will need another `make bundle` after changing.